workflow/airflow_libero/script/entrypoint.sh

```diff
@@ -2,11 +2,11 @@
 
 TRY_LOOP="20"
 
-: "${REDIS_HOST:="redis"}"
+: "${REDIS_HOST:="article-store-airflow-redis"}"
```

Comment:
these defaults are not used very much, because running an image alone will try to connect to a `redis` or `postgres` container that has not been started instead. Running the image within docker-compose will make it connect to a started `redis` or `postgres`, but will also pass `REDIS_HOST`|`POSTGRES_HOST` so it will overwrite this. Is there a more useful default value to use here? Empty? Localhost?

Reply:
You are right, I will switch them to localhost.